cmd/ocm/gcp/gcp-client-shim.go

```diff
@@ -90,42 +91,55 @@ func (c *shim) CreateWorkloadIdentityPool(
 		}
 	}
 
+	// Enable the pool if it exists but is disabled.
+	if resp.Disabled {
```

Comment:
resp != nil && resp.Disabled

Reply:
this if condition should either be incorporate into previous if\else    or  make those previous if\else return after those conditions are processed sucessfully. This just seemed mixed up

Reply:
I have simplified the conditional logic of this method based on your comment.